client/src/containers/section/map/attribution.tsx

```diff
@@ -0,0 +1,28 @@
+import { useRecoilValue } from 'recoil';
+
+import { mapSettingsAtom } from '@/store/index';
+
+import { useTheme } from '@/hooks/ui/theme';
+
+const Attribution = () => {
+  const { basemap } = useRecoilValue(mapSettingsAtom);
+  const theme = useTheme('text');
+  const attributionLinkText = {
+    'basemap-satellite': { text: 'ESRI', href: 'https://www.esri.com/home' },
+    'basemap-light': { text: 'Carto', href: 'https://carto.com/' },
```

Comment:
Those are not the correct attributions. Have a look at [ORC-81](https://vizzuality.atlassian.net/browse/ORC-81).

Also, please note that OSM Liberty needs further [attributions](https://github.com/maputnik/osm-liberty/blob/gh-pages/LICENSE.md#design-license):
>[© OpenMapTiles](https://openmaptiles.org/) [© OpenStreetMap contributors](https://www.openstreetmap.org/copyright)

Reply:
Do we need the Powered by?